docs/synapse/userguides/storm_ref_type_specific.ipynb

```diff
@@ -1785,20 +1789,46 @@
   {
    "cell_type": "raw",
    "metadata": {},
+   "source": [
+    "- Set the observed time window using a time zone offset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Define and print test query\n",
+    "q = '[ inet:ipv4=1.2.3.4 +#cno.infra.sink.hole=(2017/06/13 09:46+04:00, \"?\") ]'\n",
+    "print(q)\n",
+    "# Execute the query and test\n",
+    "podes = await core.eval(q, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
    "source": [
     "Despite storing ``time`` types as epoch millis, Storm does not accept millis format as direct input. For example, it is not possible to input a time value in the format ``1544953072324`` (Storm will attempt to interpret that as the string ``YYYYMMDDhhmmssmmmm`` and return an error).\n",
     "\n",
     "Standard rules regarding :ref:`whitespace` and :ref:`literals` apply. For example, ``\"2018/12/16 09:37:52.324\"`` needs to be entered in single or double quotes, but ``2018/12/16`` does not. Similarly, relative times and the special time value ``?`` need to be placed in single or double quotes.\n",
     "\n",
     ".. NOTE::\n",
-    "  Synapse does not support the storage of an explicit time zone with a time value (i.e., PST, +0500). Synapse is intended to store time values in UTC for consistency, so users will need to convert date / time values to UTC before entering them and convert values to UTC during any automated ingest.\n",
+    "  Synapse does not support the storage of an explicit time zone with a time value (i.e., PST, +0500). Synapse is intended to store time values in UTC for consistency, so users will need to convert date / time values to UTC before entering them and convert values to UTC during any automated ingest. Alternatively, the time zone can be provided in +/-hh:mm format, which will then be used to convert the time to UTC for storage.\n",
```

Comment:
Probably get to remove the "+0500" from the "does not support" examples now?

Reply:
I read this as "we don't store the time zone itself", which is still true since we are just converting to utc and discarding whatever the original time zone was.  I'll try to reword though.